examples/js/loaders/MTLLoader.js

```diff
@@ -358,6 +359,22 @@ THREE.MTLLoader.MaterialCreator.prototype = {
 
 			return baseUrl + url;
 		};
+		
+		var setMapForType = function ( mapType, value ) {
```

Comment:
@mrdoob changed. I thought there was a preference for this style of declaring a function (looking at the `resolveURL` function right above it)